com.unity.render-pipelines.high-definition/Editor/Material/Decal/ShaderGraph/DecalSubTarget.cs

```diff
@@ -28,7 +28,7 @@ sealed partial class DecalSubTarget : HDSubTarget, ILegacyTarget, IRequiresData<
         protected override string customInspector => "Rendering.HighDefinition.DecalGUI";
         protected override string renderType => HDRenderTypeTags.Opaque.ToString();
         protected override string renderQueue => HDRenderQueue.GetShaderTagValue(HDRenderQueue.ChangeType(HDRenderQueue.RenderQueueType.Opaque, decalData.drawOrder, false, false));
-        protected override ShaderID shaderID => HDShaderUtils.ShaderID.SG_Lit;
+        protected override ShaderID shaderID => HDShaderUtils.ShaderID.SG_Decal;
```

Comment:
Arf. Guess it mean that already converted material will not work, as we can't detect they are decal?

Mean for the migration step we should maybe not test against HDShaderUtils.ShaderID.SG_Decal; but just migrate every material ?

Reply:
or is this property not serialize and just a on the fly thing?

Reply:
It's not serialized, so it's fine. We never tested the shader id in our upgrade script anyway